added clustering_evaluation_metrics and updated the readme file #5

guntaaskapoor24 opened the pull request:
The clustering_evaluation_metrics notebook includes various evaluation metrics like Silhouette Score, Davies Bouldin Index (DBI) and Adjusted Rand Index (ARI).

guntaaskapoor24 committed:
added clustering_evaluation_metrics

This notebook includes various clustering evaluation metrics like Silhouette Score, Davies Bouldin Index and Adjusted Rand Index. This notebook is useful for evaluating the performance and the robustness of the Clustering Algorithms.

guntaaskapoor24 committed:
Update README.md

updated readme file

guntaaskapoor24 commented:
@Cgarg9 Hi! Resolved Issue #5

Cgarg9 commented:
@guntaaskapoor24 the results for ARI need to be added only theory is there right now.
Also why was PCA used for visualisation?

guntaaskapoor24 committed:
Delete clustering/clustering_evaluation_metrics.ipynb

guntaaskapoor24 committed:
added clustering_evaluation_metrics

This notebook aims to explain the various clustering evaluation techniques like Silhouette Score, Davies - Bouldin Index and Adjusted Rand Index and how to calculate these.

guntaaskapoor24 commented:
@Cgarg9 Hi! Updated the notebook to measure ARI and explain why PCA was used.

Cgarg9 commented:
@guntaaskapoor24 looks great. Merging
